### PR TITLE
Update azip-integration.md

### PR DESCRIPTION
**Pull request by tareqmusmar**
This is not documented but is a known issue

### CODE REVIEW

File: CloudAppSecurityDocs/azip-integration.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ You can see the sensitivity labels from Microsoft Purview Information Protection
 > - Defender for Cloud Apps doesn't support removing labels with protection from files that were labeled outside Defender for Cloud Apps.
 > - Defender for Cloud Apps doesn't support reading labels of password-protected files.
 > - Empty files will not be labeled.
+> - Defender for Cloud Apps doesn't support labeling files in [library that is configured to require check out](https://support.microsoft.com/en-us/office/set-up-a-library-to-require-check-out-of-files-0c73792b-f727-4e19-a1f9-3173899e695b).
```
Comment (Court72) [inline CloudAppSecurityDocs/azip-integration.md:63]:
```suggestion
> - Defender for Cloud Apps doesn't support labeling files in a [library that is configured to require checkout](https://support.microsoft.com/office/set-up-a-library-to-require-check-out-of-files-0c73792b-f727-4e19-a1f9-3173899e695b).
```

**Comment by Court72** [comment]
@dcurwin 

Can you review the proposed changes? I suggested a grammatical edit and removing the `en-us` from the new link.

*IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger5** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dcurwin** [comment]
I will check to see if this change is approved.

**Comment by dcurwin** [comment]
@tareqmusmar  - Thanks for your contribution. We've published your note.

**Pull request merged**